README.md

```diff
@@ -1,29 +1,38 @@
+<div align="center">
+<img src="https://raw.githubusercontent.com/AMereBagatelle/fabricskyboxes/1.19.x-dev/src/main/resources/assets/fabricskyboxes/fabricskyboxes_logo.png" width="150">
+
 # FabricSkyboxes
 
-##### Implements Custom Skyboxes, like Optifine! (With a different format)
+#### Implements custom skyboxes, like OptiFine! (With a different format)
 
 ![](https://cdn-raw.modrinth.com//data/YBz7DOs8/images/49eab46d1e74e8f781404a8f773a1a09dedda5aa.png)
 _Screenshot taken with the [Awesome Skies](https://github.com/heymanMC/Awesome-Skies) resource pack, a pack made to run **only** under FSB_ (Credit: [heyman](https://github.com/heymanMC))
 
+</div>
+
 ## Purpose
 
-This mod's purpose is to allow the specification of custom Skyboxes, with any textures that you may want!
+This mod's purpose is to allow the specification of custom skyboxes, with any textures that you may want!
 
 ## Usage
 
-### FSB Skybox format
+### FSB skybox format
 
 The documentation on this format can be found [here](docs).
 
-### OptiFine Skybox format
+### OptiFine skybox format
```

Comment:
I believe `OptiFine` could be changed to `Optifine` here too, and throughout the document.

Reply:
But `OptiFine` is the correct capitalization

Reply:
Huh.  Thanks for letting me know!